src/TorchSharp/Optimizers/ASGD.cs

```diff
@@ -6,6 +6,8 @@
 
 namespace TorchSharp
 {
+    using System.IO;
+    using Microsoft.VisualBasic;
```

Comment:
>Microsoft.VisualBasic [](http://example.com/codeflow?start=10&length=21)

this is interesting :-)

Reply:
Must be something VS added automatically. Seems to be doing that every now and then. Kinda funny, I think.